### DISCUSSION TITLE
Please support milestoned and demilestoned events for pull request (not just issue)

### DISCUSSION THREAD

**Discussion by pllim** [community/community#8674] [Actions]
I have been waiting very long for this one. Also see https://github.community/t/feature-request-add-milestone-changes-as-activity-type-to-pull-request/16778/10

I know it is implemented for issue and pull request is a subset of issue, but it does not work when Action is triggered by `pull_request`. I contacted GitHub Help Desk about this and all they said was to watch out for their quarterly goals or whatever, with no ETA.

**Comment by ethomson** [comment]
Thanks @pllim - GitHub Actions triggers work based on events that come from other parts of the platform.  In this case, this is part of the pull request platform, so once they add this data to their event payloads we'll be able to add support for triggering based on it in GitHub Actions.  I've asked the team that looks after pull requests to take a look.

**Reply by flying-sheep** [reply]
It’s weird: The webhooks docs have it but the workflows docs don’t: https://github.com/scverse/scanpy/pull/2518#discussion_r1234042210

does that mean that the webhooks support the event, while workflows don’t?
